javascript/atoms/test/html5/shadow_dom_test.html

```diff
@@ -175,12 +184,7 @@ <H4>Younger Child Shadow</H4>
     var parentHeading = /** @type {!Element} */ (document.querySelector(
       "* /deep/ #parentHeading"));
     var oldChildDiv = document.querySelector("* /deep/ #olderChildDiv");
-    var neverShownDiv = /** @type {!Element} */ (document.querySelector(
-      "* /deep/ #neverShownDiv"));
-  
-    // neverShown is in an older shadow DOM, and thus isn't displayed
-    assertFalse(bot.dom.isShown(neverShownDiv));
```

Comment:
FYI i removed this test because AFAIK its invalid per the old spec and doesn't apply in the new spec (as `createShadowRoot` has gone).

this test relies on the fact chrome once allowed you to create a shadow root multiple times, which was removed shortly after. now it just throws an exception or doesn't do anything.